txbits/app/wallet/Wallet.scala

```diff
@@ -304,6 +358,7 @@ class Wallet(rpc: JsonRpcHttpClient, currency: CryptoCurrency, nodeId: Int, para
   }
 
   private def getBalance = {
+    // This method is unused as balance is now tracked in the database
```

Comment:
I don't like the idea of automatically adjusting the balance. If we have less funds than we should have, it needs to be properly investigated.